indra/llcommon/llsdserialize.cpp

```diff
@@ -2220,54 +2228,67 @@ std::string zip_llsd(LLSD& data)
 // not very efficient -- creats a copy of decompressed LLSD block in memory
 // and deserializes from that copy using LLSDSerialize
 LLUZipHelper::EZipRresult LLUZipHelper::unzip_llsd(LLSD& data, std::istream& is, S32 size)
```

Comment:
I realize you inherited this function signature. But going forward, please use `size_t` where unsigned size works and `llssize` where we must be able to express negatives.